tracing-journald/src/lib.rs

```diff
@@ -150,6 +152,34 @@ impl Subscriber {
         self
     }
 
+    /// Adds fields that will get be passed to journald with every log entry.
+    ///
+    /// The input values of this function are interpreted as `(field, value)` pairs.
+    ///
+    /// This can for example be used to configure the syslog facility.
+    /// See [Journal Fields](https://www.freedesktop.org/software/systemd/man/systemd.journal-fields.html)
+    /// and [journalctl](https://www.freedesktop.org/software/systemd/man/journalctl.html)
+    /// for more information.
+    ///
+    /// ```rust
+    /// # use tracing_journald::Subscriber;
+    /// let sub = Subscriber::new()
+    ///     .unwrap()
+    ///     .with_custom_fields([("SYSLOG_FACILITY", "17")]);
+    /// ```
+    ///
+    pub fn with_custom_fields<T: AsRef<str>, U: AsRef<str>>(
```

Comment:
On review, the second field doesn't actually have to be a string:
```suggestion
    pub fn with_custom_fields<T: AsRef<str>, U: AsRef<[u8]>>(
```
This should still accept `&'static str` and `String`.

Reply:
Fixed